xbmc/windowing/android/AndroidUtils.h

```diff
@@ -31,7 +31,18 @@ class CAndroidUtils : public ISettingCallback
   static const std::string SETTING_LIMITGUI;
   void OnSettingChanged(const std::shared_ptr<const CSetting>& setting) override;
 
+  // HDR info
+  static const int HDR_TYPE_DOLBY_VISION = 1;
```

Comment:
Maybe convert to enum or enum class

Reply:
`enum class` did not work for the `find` in the other methods:
`std::find(types.begin(), types.end(), CAndroidUtils::HDRTypes::HLG) != types.end();`

I didn't bother keeping it and just changed to `enum`